src/DaPigGuy/PiggyCustomEnchants/enchants/armor/ShrinkEnchant.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaPigGuy\PiggyCustomEnchants\enchants\armor;
+
+use DaPigGuy\PiggyCustomEnchants\enchants\CustomEnchant;
+use DaPigGuy\PiggyCustomEnchants\enchants\ToggleableEnchantment;
+use DaPigGuy\PiggyCustomEnchants\enchants\traits\ReactiveTrait;
+use DaPigGuy\PiggyCustomEnchants\enchants\traits\TickingTrait;
+use pocketmine\event\Event;
+use pocketmine\event\player\PlayerToggleSneakEvent;
+use pocketmine\inventory\Inventory;
+use pocketmine\item\Item;
+use pocketmine\Player;
+use pocketmine\utils\TextFormat;
+
+/**
+ * Class ShrinkEnchant
+ * @package DaPigGuy\PiggyCustomEnchants\enchants\armor
+ */
+class ShrinkEnchant extends ToggleableEnchantment
+{
+    use ReactiveTrait;
+    use TickingTrait;
```

Comment:
Why not ``use ToggleTrait`` aswell? The ToggleableEnchantment functions are no longer overwritten here, is it still needed?

Reply:
Because ToggleableEnchantment uses ToggleTrait